### DISCUSSION TITLE
Highlight failed tasks on summary when Error handler is defined

### DISCUSSION THREAD

**Discussion by pitermarx** [cake-build/cake#3378] [Ideas]
If I define an error handler with .ContinueOnError() or .OnError() the task execution will continue, but the summary wont show any info about that.
What if we highlighted in red (or smth else) the summary for failed tasks?
![image](https://user-images.githubusercontent.com/3081298/119803482-67950e80-bed7-11eb-903b-3b3e75378188.png)

**Comment by devlead** [comment]
While not possible now that could be a could addition.

What you can do now is to have a Task that
* Depends on `IntegrationTests`
* With a criteria that's set before exiting  `IntegrationTests`

That would result in an output like this
![image](https://user-images.githubusercontent.com/1647294/119804652-d62a9a00-bee0-11eb-8ea6-d470717e641f.png)

And if you run with Verbose logging it could look something like this
![image](https://user-images.githubusercontent.com/1647294/119804807-fa867680-bee0-11eb-9e2f-2dc6638f8608.png)

Example script for above
```cake
public class BuildData
{
    public bool IntegrationTestsSuccessful { get; set; }
}

Setup( setupContext => new BuildData() );

Task("Clean")
    .Does(() => {});

Task("Restore")
    .IsDependentOn("Clean")
    .Does(() => {});

Task("Build")
    .IsDependentOn("Restore")
    .Does(() => {});

Task("BuildDb")
    .IsDependentOn("Build")
    .Does(() => {});

Task("SetupTestDatabases")
    .IsDependentOn("BuildDb")
    .Does(() => {});

Task("IntegrationTests")
    .IsDependentOn("SetupTestDatabases")
    .ContinueOnError()
    .Does<BuildData>((context, data) => {
    throw new Exception("sdds");
    // Never reach here
    data.IntegrationTestsSuccessful = true;
});

Task("ValidateIntegrationTests")
    .IsDependentOn("IntegrationTests")
    .WithCriteria<BuildData>(
        (context, data) => data.IntegrationTestsSuccessful,
        "ValidateIntegrationTests skipped because non-sucessful IntegrationTests"
        )
    .Does(() => {});

Task("Default")
    .IsDependentOn("ValidateIntegrationTests");


RunTarget(Argument("Target", "Default"));
```

**Comment by pitermarx** [comment]
made a PR
https://github.com/cake-build/cake/pull/3379

**Comment by augustoproiete** [comment]
:tada: This has been released as part of Cake v1.3.0 :tada:

The release is available on:

- [GitHub Release](https://github.com/cake-build/cake/releases/tag/v1.3.0)
- [.Net Global Tool](https://www.nuget.org/packages/Cake.Tool/v1.3.0)
- [NuGet Package](https://www.nuget.org/packages/Cake/v1.3.0)
- [Chocolatey Package](https://chocolatey.org/packages/cake.portable/v1.3.0)
